.jenkins/infrastructure/docker/dockerfiles/linux/Dockerfile

```diff
@@ -64,6 +62,25 @@ ARG devkits_uri
 # Check to make sure devkits_uri is defined as build arg
 RUN test ! -z ${devkits_uri+x}
 
+# Install essential packages
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get -y install make build-essential git jq vim curl wget netcat apt-transport-https unzip
+
+# Add Microsoft repo
+RUN echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ bionic main" | tee /etc/apt/sources.list.d/azure-cli.list
+RUN wget https://packages.microsoft.com/keys/microsoft.asc
+RUN apt-key add microsoft.asc
+RUN apt-get update
+
+# Install Microsoft packages
```

Comment:
/nit

Technically you are installing only Azure CLI and not Microsoft packages :)